android-agent/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -370,7 +372,9 @@ private void applyConfiguration() {
             if (initializationEvents == InitializationEvents.NO_OP) {
                 initializationEvents = new SdkInitializationEvents();
             }
-            initializationEvents.recordConfiguration(config);
+            Map<String, String> configMap = new HashMap<>();
+            // TODO: Convert config to map
```

Comment:
I opened https://github.com/open-telemetry/opentelemetry-android/issues/277 to track this.

Reply:
I wish there could be a way to pass the config object without having to transform it into a map though, it feels awkward. But I guess we can leave it as is for now. I'll follow up on the ticket on this.